ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -578,70 +578,85 @@ public void deleteErrorCasesTest() {
   }
 
   /**
-   * Test DELETE authorization success for {@link BlobStore#delete(MessageWriteSet)}.
-   * If either accountId or containerId is -1, validation is skipped.
+   * Test DELETE with right accountId and containerId.
+   * If BlobId version is 1, accountId and containerId validation is skipped.
    */
   @Test
   public void deleteAuthorizationSuccessTest() throws Exception {
     short[] accountIds = {-1, Utils.getRandomShort(TestUtils.RANDOM), -1};
     short[] containerIds = {-1, -1, Utils.getRandomShort(TestUtils.RANDOM)};
     for (int i = 0; i < accountIds.length; i++) {
-      MockId mockId = put(1, PUT_RECORD_SIZE, Utils.Infinite_Time, accountIds[i], containerIds[i]).get(0);
-      delete(
-          new MockId(mockId.getID(), Utils.getRandomShort(TestUtils.RANDOM), Utils.getRandomShort(TestUtils.RANDOM)));
+      // put and get version 1 blobId, acoountId and containerId check is skipped.
```

Comment:
not required  - impl detail. Just check for success when `true` and failure when `false`

Reply:
fixed.